src/Main.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Scanner;
+
+/**
+ * Created by Nigel on 6/16/16.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+        int id = 0;
+        int choice = 0;
+        Inventory theInventory = new Inventory();
+
+        do {
+            switch (theInventory.menuOptions()) {
+                case 1:
+                    id++;
+                    theInventory.addItem(id);
+                    break;
+                case 2:
```

Comment:
LOTS of repeat code here.  You could probably just put the if/else statement before the switch so that it either prints the inventory or lets the user know that there isn't anything in the list.